### ISSUE TITLE
onSuggestionsFetchRequested req.reason doesn't exist in 9.1

### ISSUE THREAD

**Issue by lvpro** [moroshko/react-autosuggest#386]
```
  onSuggestionsFetchRequested = (req: Autosuggest.SuggestionsFetchRequest) => {
    // req.reason seems to no longer exist in 9.1.x
    this.getSuggestionsDebounced(req.value);
  }
```
The object given to onSuggestionsFetchRequested contains only 1 key: value.  reason seems to be missing. :-/

**Comment by moroshko** [comment]
If you check [the docs of 9.1.0](https://github.com/moroshko/react-autosuggest/tree/v9.1.0#onSuggestionsFetchRequestedProp), you'll see that `reason` is not there.
https://github.com/moroshko/react-autosuggest/pull/385 was merged yesterday, and will hopefully be released soon.

**Comment by moroshko** [comment]
@lvpro `reason` in available now is [v9.2.0](https://github.com/moroshko/react-autosuggest/releases/tag/v9.2.0).
